add more info around custom partition_size (#30)
ArweaveTeam/docs.arweave.org-info: mining/mining-guide.md

To maximize your mining efficiency, store data on 4 TB hard drives capable of reading about 200 MiB/s. It is fine to use faster disks, but the extra cost won't be justified.

> The first step is to create a folder inside `[data_dir]/storage_modules/` for each of the partisions that you intend to mine with. Partition folder names should use the following pattern: `storage_module[_partition_size]_[partition_index]_[your_mining_address]`. As you will see further, specifying `partition_size` is optional. The default partition size is 3.6 TB, as Arweave dataset is logically partitioned into collections of 3.6 TB "mining partitions". For any given `partition_size`, you should take into account around 10% metadata overhead (such as merkle proofs). Hence, for the default 3.6 TB `partition_size`, we recommend a 4 TB disk. The partitions are indexed sequentially starting from 0, with 0 being the very first 3.6 TB (or `partition_size`) worth of data stored on Arweave, and ranging to whatever the current mining partition count is. You can choose which mining partitions you store by indicating their `partition_index` in the folder name. 

Reviewer: ```suggestion
The Arweave dataset is logically partitioned into collections of 3.6 TB "mining partitions". You will store some or all of those mining partitions on your miner in "storage modules". Storage modules can be any size (smaller or larger than the default 3.6TB mining partition size), but many miners opt to align their storage modules with the Arweave partitions.

To setup your storage modules, the first step is to create a folder inside `[data_dir]/storage_modules/` for each of the storage modules that you intend to mine with. Storage module folder names should use the following pattern: `storage_module[_storage_module_size]_[storage_module_index]_[your_mining_address]`. The default storage module size is 3.6TB - in that case specifying `storage_module_size` is optional.

For any given `storage_module_size`, you should allow for an additional 10% metadata overhead (such as merkle proofs). This is why, for the default 3.6 TB `storage_module_size`, we recommend a 4 TB disk. The storage modules are indexed sequentially starting from 0, with 0 being the very first 3.6 TB (or `storage_module_size`) worth of data stored on Arweave, and ranging up to or beyond the current Arweave dataset size (`weave_size`). You can choose which mining partitions you store by indicating their `storage_module_index` in the folder name. 
```

Reviewer: Sorry for all the edits. The distinction between storage module and partition is awkward but important... which I think is why I've put off updating the docs.

> For example, to set up a storage module with the very first mining partition in the weave with the default 3.6 TB `partition_size` (packed with your mining address), create the folder such as `[data_dir]/storage_modules/storage_module_0_[your_mining_address]`. If you were to store 2 TB Partitions, create the folder such as: `[data_dir]/storage_modules/storage_module_2000000000000_0_[your_mining_address]`.

Reviewer: ```suggestion
For example, to set up a storage module with the very first mining partition in the weave with the default 3.6 TB `storage_module_size` (packed with your mining address), create the folder such as `[data_dir]/storage_modules/storage_module_0_[your_mining_address]`. If you were to store 2 TB storage modules, create the folder such as: `[data_dir]/storage_modules/storage_module_2000000000000_0_[your_mining_address]`.
```

After creating the relevant folders for your chosen partitions, mount your drives onto them. E.g.,
```
sudo mount /dev/sda [data_dir]/storage_modules/storage_module_0_[your_mining_address]

> sudo mount /dev/sda [data_dir]/storage_modules/storage_module_2000000000000_0_[your_mining_address] // custom 2 tb partition size

Reviewer: ```suggestion
sudo mount /dev/sda [data_dir]/storage_modules/storage_module_2000000000000_0_[your_mining_address]
```

- Having two or more storage modules that store the same mining partition (say, the partition at index 0 more than once) with the same mining address does not increase your mining performance. Also, it is more profitable mine a complete replica (all mining partitions) of the weave packed with a single address than mine off an equal amount of data packed with different mining addresses. Currently, we only support one mining address per node.
- If you want to copy the contents of a storage module elsewhere, restart the node without the corresponding `storage_module` command line parameter, copy the data, and restart the node with the `storage_module` parameter again. You can attach the copied data as a storage module to another node. Just make sure to not copy while the node is interacting with this storage module. Do NOT mine on several nodes with the same mining address simultaneously (see the warning below.)
- Make sure the disks with the storage modules have sufficient available space for both the data iteself and metadata (10% of the size of the data). Note that `disk_space` command line parameter does NOT apply to the storage modules.

> - If you created storage modules with custom `partition_size` as mentioned above, make sure to specify the `partition_size` in your command line invocation as follow:\

Reviewer: ```suggestion
- If you created storage modules with custom `storage_module_size` as mentioned above, make sure to specify the `storage_module_size` in your command line invocation as follows:
```

>   `storage_module [partition_index],[partition_size],[your_mining_address]`\

Reviewer: ```suggestion
  `storage_module [storage_module_index],[storage_module_size],[your_mining_address]`\
```

>   The module will sync data with the weave offsets between `partition_index * partition_size` (in bytes) and `(partition_index + 1) * partition_size` at folder `[data_dir]/storage_modules/storage_module[_partition_size]_[partition_index]_[your_mining_address]`.

Reviewer: ```suggestion
  The module will sync data with the weave offsets between `storage_module_index * storage_module_size` (in bytes) and `(storage_module_index + 1) * storage_module_size` at folder `[data_dir]/storage_modules/storage_module[_storage_module_size]_[storage_module_index]_[your_mining_address]`.
```
